dts/bindings/serial/st,stm32-uart-bm.yaml

```diff
@@ -0,0 +1,27 @@
+description: STM32 UART
+
+compatible: "st,stm32-uart-bm"
+
+include: [uart-controller.yaml, pinctrl-device.yaml]
+
+properties:
+    reg:
+      required: true
+
+    interrupts:
+      required: true
+
+    single-wire:
```

Comment:
I think this single-wire mode and any functionality related to it could/should be removed, since our driver and PHY are not designed to support this modality.